### PR TITLE
Create ui-c8y-1021-0-0-enable-edit-mode-when-freshly-created-dashboard-is-opened-7151.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre-c8y

Ticket: [MTM-61183](https://cumulocity.atlassian.net/browse/MTM-61183)
Original PR: [7151](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7151)

[MTM-61183]: https://cumulocity.atlassian.net/browse/MTM-61183?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-enable-edit-mode-when-freshly-created-dashboard-is-opened-7151.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Enable edit mode when freshly created dashboard is opened. (#7151)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-enable-edit-mode-when-freshly-created-dashboard-is-opened-7151.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Enable edit mode when opening a newly created dashboard
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-enable-edit-mode-when-freshly-created-dashboard-is-opened-7151.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61183
+version: 1021.0.0
+---
+Enable edit mode when freshly created dashboard is opened. (#7151)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-enable-edit-mode-when-freshly-created-dashboard-is-opened-7151.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when creating a new dashboard, users had to manually enable the edit mode to start customizing the dashboard. With this change, the edit mode is now automatically enabled when opening a newly created dashboard. This improvement allows users to immediately begin editing and configuring their dashboards without the need for an additional step, enhancing the overall user experience and streamlining the dashboard creation process.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2489

**Pull request closed without merging**